rust-manifest/src/config.rs

```diff
@@ -9,19 +10,7 @@ pub const DEFAULT_CONFIG_VERSION: &'static str = "1";
 #[derive(Clone, Debug)]
 pub struct Config {
     pub config_version: String,
-    pub remote: Option<ConfigRemote>,
-    pub install: ConfigInstall,
```

Comment:
OK, no problem - can re-add them later as required if the intent was not to remove them permanently. The remote tells it how to update, ie. where to find newer versions of the manifest. For non-custom toolchains managed by multirust, it's not strictly necessary since it can figure it out from the toolchain name. However, for custom toolchains, or independent installations, it needs to know what it should track (eg. i686-pc-windows-msvc beta) and where to look for it (eg. `http://static.rust-lang.org/dist/`)

By allowing an arbitrary URL, it allows a great deal of flexibility - you could quite easily set up a custom rust distribution with extra extensions, or have a local dist server set up to avoid needing internet access.